### DISCUSSION TITLE
The home bandwidth used is 1000M, and the bandwidth of the entire line is 1000M , but the target bit of the self-built rustdesk server is only 1368kb.

### DISCUSSION THREAD

**Discussion by manyouzhe** [rustdesk/rustdesk#8231] [General]
![Quicker_20240601_084500](https://github.com/rustdesk/rustdesk/assets/35192593/0e6fa957-544d-4ea2-93a9-06f3b6e6a678)
I built my own rustdesk server for remote operation and file transfer. The home bandwidth used is 1000M, and the bandwidth of the entire line is 1000M (the maximum transmission speed I verified by other means is 100M), but the target bit of the self-built rustdesk server is only 1368kb. Is there any way to increase this target bit? Thank you!

**Comment by 21pages** [comment]
![54076dac558e3fec642cc3169d79feb](https://github.com/rustdesk/rustdesk/assets/14891774/6a5550d1-8f3a-42f3-97ff-9b7068b4c935)

**Reply by manyouzhe** [reply]
Thank you very much for your reply. This method does change the destination bit, but the file transfer speed does not change.

**Reply by xlionjuan** [reply]
@manyouzhe change this env for hbbr

https://github.com/rustdesk/rustdesk-server/discussions/371#discussioncomment-8511523
